js/Utils.js

```diff
@@ -115,3 +115,18 @@ function capitalize(s){
 function removeTrailingZeros(value) {
     return (value + '').replace(/(\.\d*?)0+$/, '$1').replace(/\.$/, '');
 }
+
+function parseDateStringOrRestoreHeightAsInteger(value) {
+    // Parse date string or restore height as integer
+    var restoreHeight = 0;
+    if (value.indexOf('-') === 4 && value.length === 10) {
+        restoreHeight = Wizard.getApproximateBlockchainHeight(value, Utils.netTypeToString());
+    } else if (value.substring(0, 4) >= "2014" && value.substring(0, 4) <= "2025" && value.length === 8) {
```

Comment:
```suggestion
    } else if (parseInt(value.substring(0, 4)) >= 2014 && parseInt(value.substring(0, 4)) <= 2025 && value.length === 8) {
```

This isn't 100% necessary but I would add this to avoid bugs like this https://stackoverflow.com/questions/5630123/javascript-string-integer-comparisons

I know in this specific code this can't happen but let's be on the safe side.